src/Microsoft.ML.Core/Data/ColumnTypeExtensions.cs

```diff
@@ -22,5 +22,44 @@ internal static class ColumnTypeExtensions
         /// Zero return means either it's not a key type or the cardinality is unknown.
         /// </summary>
         public static int GetKeyCount(this ColumnType columnType) => (columnType as KeyType)?.Count ?? 0;
+
+        public static ColumnType ItemType(this ColumnType columnType) => (columnType as VectorType)?.ItemType ?? columnType;
```

Comment:
it deserves comment.

Reply:
Agreed. I'll add it. Plus I need to make these methods have "verb" names.